win32/perllib.c

```diff
@@ -67,6 +68,13 @@ win32_checkTLS(PerlInterpreter *host_perl)
         }
         host_perl->Isys_intern.cur_tid = tid;
     }
+#else
```

Comment:
Would this handle the non-ithreads multiplicity case? (though I doubt that really exists on Windows).

Reply:
My test builds are with:
```
gmake CCTYPE=MSVC143 USE_ITHREADS=undef ...
```
which results in:
```
C:\...\perl\win32>..\perl -I..\lib -V:usemultiplicity
usemultiplicity='define';
```
Unfortunately we don't seem to have any active Win32 smoke testers.